src/data/number/PLNumber.ts

```diff
@@ -72,12 +72,16 @@ export class PLNumber
   }
 
   public divide(d: PLNumber): PLNumber {
+    var quotentDecimals = MAX_DECIMALS
```

Comment:
please never use `var`, you should use `let` or `const`

Reply:
@maxinteger Done. Can you pls take a look?